src/machines/java/com/enderio/machines/client/rendering/travel/TravelAnchorRenderer.java

```diff
@@ -46,6 +48,9 @@ public void render(AnchorTravelTarget travelData, LevelRenderer levelRenderer, P
 
         // Render Model
         BlockState blockState = minecraft.level.getBlockState(travelData.getPos());
+        if (blockState.is(MachineBlocks.PAINTED_TRAVEL_ANCHOR)) {
```

Comment:
For future compatibility sakes, we could change this to getting the block entity, then doing an is PaintedTravelAnchorBlockEntity check :)